lib/PlacesProvider.js

```diff
@@ -222,8 +222,8 @@ Links.prototype = {
     }
 
     // setup order by clause
-    let orderbyClause = (order === "frecency") ?
-                    "ORDER BY frecency DESC, lastVisitDate DESC, url" :
+    let orderbyClause = (order === "bookmarksFrecency") ?
+                    "ORDER BY bookmarkDateCreated DESC, frecency DESC, lastVisitDate DESC, url" :
```

Comment:
Comment for the function parameters has been obsoleted.

We'll also want tests for ordering.